dashboard/db/migrate/20230522232402_create_lti_user_identities.rb

```diff
@@ -0,0 +1,12 @@
+class CreateLtiUserIdentities < ActiveRecord::Migration[6.1]
+  def change
+    create_table :lti_user_identities do |t|
+      t.string :subject, null: false
+      t.references :lti_integration, null: false, foreign_key: true, type: :bigint
+      t.references :user, null: false, foreign_key: true, type: :integer
```

Comment:
Any reason why you re-ordered the columns relative to the Tech Spec? https://docs.google.com/document/d/1tTwAF40cnn2oUvK2CmRO-a5qYXkCR5rR5gBJS2-zV_U/edit#heading=h.ulde8t7jj1tf

Reply:
Not particularly. I'm not sure they are ordered well in the tech spec though? We'll usually query these on `subject`, and it's the only non-foreign key column, so it felt like it should come first. The other two are 🤷‍♂️

Reply:
Is subject going to be the same ID from the `sub` claim from the JWT?

Reply:
Yep. Initially I had this as `lti_user_id`, but figured aligning with the `subject` from the JWT was more clear.